metrics_handler/main.py

```diff
@@ -42,6 +42,7 @@
 BQ_METRIC_TABLE_NAME = 'metric_history'
 METRICS_WRITTEN_TOPIC = 'metrics-written'
 MIN_MSG_AGE_SEC = 60
+MAX_SEC_TO_WAIT_FOR_MESSAGE = 48 * 60 * 60
```

Comment:
nit: Can you add a comment with the conversion to hours?

Reply:
Done